blockbase/theme.json

```diff
@@ -339,8 +332,6 @@
 			]
 		},
 		"typography": {
-			"customFontSize": true,
```

Comment:
Are you sure we can remove this? I didn't see it mentioned in the GB PR?

Reply:
https://github.com/WordPress/gutenberg/pull/36246 yeah I think it is in there

Reply:
Oh sorry I was looking at the wrong PR